oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -494,7 +494,9 @@ bool checkFruPresence(const char* objPath)
         isPresent = std::get<bool>(propVal);
     }
     catch (const sdbusplus::exception::SdBusError& e)
-    {}
+    {
+        error("Failed to call the D-Bus Method ERROR={ERR}", "ERR", e.what());
```

Comment:
Can you change something like "D-Bus method call to get the FRU presence failed" ?

Reply:
updated as per comment.